common/src/main/webapp/sass/datepicker.scss

```diff
@@ -1,14 +1,22 @@
 @use "react-datepicker/src/stylesheets/datepicker.scss";
 
 .react-datepicker {
-  font-family: "Lucida Grande", "Lucida Sans Unicode", Arial, Helvetica, sans-serif;
+  font-family: "Lucida Grande", "Lucida Sans Unicode", Arial, Helvetica,
```

Comment:
Should we use the same font as the system?

Reply:
Good idea. I actually changed to `inherit` so that it's consistent with the rest of the UI.